app/app/urls.py

```diff
@@ -72,6 +72,7 @@
 
     # action URLs
     re_path(r'^bounty/quickstart/?', dashboard.views.quickstart, name='quickstart'),
+    url(r'^bounty/request/?', dashboard.views.new_bounty_request, name='new_request'),
```

Comment:
Since we are using Django 2, and since in Django 2, `url` is just an alias of `re_path` and is likely to be deprecated in future, so to avoid tech debt, we are trying to use `re_path` for all the new URLs. So it would be great you can just basically replace `url` to `re_path`. 😄